mozci/util/taskcluster.py

```diff
@@ -37,16 +38,18 @@ def _handle_artifact(path, response):
     return response
 
 
-def get_artifact_url(task_id, path):
+def get_artifact_url(task_id, path, use_community=False):
```

Comment:
I'd prefer a `root_url` parameter which is `PRODUCTION_TASKCLUSTER_ROOT_URL` by default, it's more generic